index.js

```diff
@@ -1,5 +1,18 @@
 var CoreObject     = require('core-object');
 var chalk = require('chalk');
+var merge = require('lodash.merge');
+
+function _createPluginHelper() {
```

Comment:
Ah yeh. That naming is a hangover from how I wrote it initially. Will change now